frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BasePullBranchFastForwardOnlyAction.java

```diff
@@ -157,11 +128,13 @@ private void doPullNonCurrentBranchFastForwardOnly(Project project,
             remoteBranch.getName()),
         format(getString("action.GitMachete.BasePullBranchFastForwardOnlyAction.notification.title.pull-success"),
             remoteBranch.getName())) {
-      // We can only enqueue the update of local branch once the update of remote branch is completed.
+
       @Override
       @UIEffect
       public void onSuccess() {
-        ourRemoteToOurLocalBranchFetchBackgroundable.queue();
+        String repoName = gitRepository.getRoot().getName();
+        FetchAllRemotesAction.updateLastFetchTimeForRepo(repoName);
```

Comment:
Hmm actually maybe this method could live in `FetchBackgroundable` instead? 🤔 so that `FetchAllRemotesAction` doesn't have any _special status_ over e.g. this very class

Reply:
Or even extract a completely separate class for tracking those latest fetch times